torch/csrc/api/include/torch/nativert/ModelRunnerHandle.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <string>
+#include <unique_ptr>
+#include <unordered_map>
+#include <vector>
+
+#include <ATen/core/IValue.h>
+#include <c10/macros/Export.h>
+
+namespace torch::nativert {
+
+// We don't want to forward declare in general but including ModelRunner will
+// pollute the public API namespace too much. Therefore, we just use pimpl an
+// incomplete ModelRunner here.
+class ModelRunner;
+
+class TORCH_API ModelRunnerHandle {
```

Comment:
So going to have to add every function we end up needing from nativeRT in ET desktop to these handls over time?

Reply:
yeah for now. hopefully there should be <10 of them